src/components/InfoBanner/InfoBanner.css

```
@@ -2,12 +2,17 @@
     padding-top: 1em;
     padding-bottom: 1em;
     display: flex;
-    justify-content: space-between;
+    justify-content: center;
     align-items: center;
     padding-left: 10%;
     padding-right: 10%;
     gap: 12em;
     font-size: 18px;
+
+    background-image: url('../../../public/InfobannerBackground.svg');
```

```suggestion
    background-image: url('/InfobannerBackground.svg');
```

These will make the resource be serves as a public static resource.

---

Fixed in most recent commit!